e2e/tests.js

```diff
@@ -1,11 +1,34 @@
+/*
```

Comment:
Do we want to just go with a truncated MIT license? cc @gergelyke @rtsao 

```
/** Copyright (c) [YEAR] Uber Technologies, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
```

Reply:
yeah, we will - it's on my plate, will send a pr today